<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шифратор</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::WeatherClear"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Ключ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edKey"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowse">
        <property name="text">
         <string>Выбрать Файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFileName">
        <property name="text">
         <string>Файл не выбран</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Исходный текст</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="edSource"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Выходной текст</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="edDest"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnChipher">
        <property name="text">
         <string>Зашифровать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDechipher">
        <property name="text">
         <string>Дешифровать</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QRadioButton" name="rdbColumn">
          <property name="text">
           <string>Столбцовый</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdbVigenere">
          <property name="text">
           <string>Виженер</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnChipher</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cipher()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDechipher</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>decrypt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBrowse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cipher()</slot>
  <slot>decrypt()</slot>
  <slot>loadFile()</slot>
 </slots>
</ui>
